src/clients/db/Repository.ts

```diff
@@ -47,8 +48,15 @@ export default class Repository<T extends AbstractDBModel> {
   }
 
   async save(entity: T): Promise<T> {
+    const currentTime = getTime(new Date())
```

Comment:
I really prefer saving dates in `ISOString`. It has same properties of ordering like unix timestamp but it is more readable. Can you change it to [toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) everywhere?